include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -303,7 +303,8 @@ namespace internal
     template <int  face_direction,
               bool contract_onto_face,
               bool add,
-              int  max_derivative>
+              int  max_derivative,
+              bool lex_faces = false>
```

Comment:
Could you please document the new template parameter?

Reply:
Done!